packages/block-library/src/button/edit.native.js

```diff
@@ -302,48 +210,49 @@ class ButtonEdit extends Component {
 	}
 
 	dismissSheet() {
-		this.setState( {
-			isLinkSheetVisible: false,
-		} );
+		this.onHideLinkSettings();
 		this.props.closeSettingsBottomSheet();
 	}
 
-	getLinkSettings( url, rel, linkTarget, isCompatibleWithSettings ) {
+	getLinkSettings( isCompatibleWithSettings, extractSettings ) {
```

Comment:
I see that we don't need 2 arguments here. We use it with both set to `false` or `true `:)